### DISCUSSION TITLE
What's the difference between Git and GitHub Desktop? Can I run GitHub Desktop without Git?

### DISCUSSION THREAD

**Discussion by JoaoBM1** [community/community#21958] [New to GitHub]
<p>Hey there!</p>
<p>So, recently, my Programming Teacher introduced us to Git and GitHub. He told us to first install Git followed by GitHub so we could easily manage our Projects between the group members. It’s a wonderful and much easier tool (than sending the whole code by email all the times).</p>
<p>My partner forgot to install Git but we discovered that we were able to Push, Pull and Commit the changes to each other (basically update and download the repository). So it got me thinking…</p>
<p>Why were we told to install both? Can’t we just have GitHub Desktop (since it has a much friendly interface to work it)?</p>
<p>Cheers!</p>

**Comment by biebzAri234** [comment]
<p>Can you explain that a bit more?</p>

**Comment by Wabri** [comment, marked as answer]
<p>installing github desktop requires git dependency, so it is very likely that the teacher advised you to install git first and then github desktop for this fact. Since the github desktop developers are clever they thought about including the git dependency in the github installation. Hope this solve your problem</p>

**Comment by JoaoBM1** [comment]
<p>Cheers for the answer!</p>
<p>Thank you!</p>

**Comment by Wabri** [comment]
<p>Your are welcome :smiley: you have to set my answer as a solution to help other people with your same problem!</p>
